src/AutoRest.CSharp/Mgmt/Output/ResourceData.cs

```diff
@@ -26,7 +26,9 @@ public ResourceData(ObjectSchema schema, string? name = default, string? nameSpa
 
         protected override string CreateDescription()
         {
-            return BuilderHelpers.EscapeXmlDescription($"A class representing the {_clientPrefix} data model.");
+            return BuilderHelpers.EscapeXmlDescription($"A class representing the {_clientPrefix} data model.") + (string.IsNullOrWhiteSpace(ObjectSchema.Language.Default.Description) ?
+                string.Empty :
+                $"\n{BuilderHelpers.EscapeXmlDescription(ObjectSchema.Language.Default.Description)}");
```

Comment:
nit: use `Environment.NewLine`?

Reply:
Will do - and I planned to do a thorough refactor about all this when we move the escape from "when we build the descriptions" to "when we write the descriptions"